scripts/deploy/28_samb_treasury_vault.ts

```diff
@@ -36,7 +36,7 @@ const main = async ({
   console.log(`VaultLifeCycleTreasuryBare @ ${lifecycleAddress}`);
 
   const RibbonTreasuryVault = await ethers.getContractFactory(
-    "RibbonTreasuryVault",
+    "RibbonTreasuryVaultBare",
```

Comment:
Since we also reference `RibbonTreasuryVaultBare` here, if we chose to remove `RibbonTreasuryVaultBare` to prevent similar copies we can add a comment explaining what this is and when/why it should be used.